tests/e2e/test_spyre_basic.py

```diff
@@ -247,3 +247,72 @@ def test_full_batch_scheduling(model: str, backend: str, monkeypatch):
     schedule = scheduler.schedule()
 
     assert len(schedule.scheduled_new_reqs) == batch_size
+
+
+@pytest.mark.multi
+@pytest.mark.parametrize("model", get_spyre_model_list())
+@pytest.mark.parametrize("prompts", [[
+    "Provide a list of instructions for preparing"
+    " chicken soup for a family of four.", "Hello",
+    "What is the weather today like?", "Who are you?"
+]])
+@pytest.mark.parametrize(
+    "warmup_shapes",
+    [[(64, 20, 4)]])  #,[(64,20,8)],[(128,20,4)],[(128,20,8)]])
+# (prompt_length/new_tokens/batch_size)
+@pytest.mark.parametrize("tp_size", [2, 4, 8])
+@pytest.mark.parametrize(
+    "backend", [b for b in get_spyre_backend_list() if "eager" not in str(b)])
+def test_output_multi(
```

Comment:
I don't think we actually need a separate test for this though, right? Can we parameterize the existing one like this:
```
@pytest.mark.parametrize("tp_size", [param(1), param(2, marks=multi) ...])
```